serveradmin/common/templates/base.html

```diff
@@ -76,5 +80,6 @@
 <script src="{{ STATIC_URL }}js/serveradmin.js"></script>
 <script src="{{ STATIC_URL }}js/plugins/terminal_keyboard.js"></script>
 {% block additional_scripts %}{% endblock %}
```

Comment:
How much sense does it make to also include any additional stuff to the caching too? If we don't have that many additional stuff it might be ok.

Reply:
We have quite some for example in the Servershell and Resources page and we never know what will be added:

- https://github.com/innogames/serveradmin/blob/master/serveradmin/servershell/templates/servershell/index.html#L7
- https://github.com/innogames/serveradmin/blob/master/serveradmin/resources/templates/resources/index.html#L8

Also I think there is value in including these because it guarentess that browser download the latest version if something has changed in there.

Reply:
Same as above. Quite a lot in my opinion and if it were only for making sure that users get the latest version when something changes (Browser Cache).

- https://github.com/innogames/serveradmin/blob/master/serveradmin/servershell/templates/servershell/index.html#L170

Reply:
Uhh, there is even dynamic javascript in there. This should result in quite some amount of cache files. I would suggest to monitor that if that could result in a problem. Good for me to try out though.

Reply:
Why would this be a problem ? 

If you think in terms of size: We have maybe 2 handful of pages and the Javascript and CSS files are way less than a 1 MB. Every log file we have is bigger in size.